python/ray/autoscaler/resource_demand_scheduler.py

```diff
@@ -90,14 +91,9 @@ def calculate_node_resources(
 
         def add_node(node_type, available_resources=None):
             if node_type not in self.node_types:
-                logger.warn(
-                    f"Missing entry for node_type {node_type} in "
-                    f"cluster config: {self.node_types} under entry "
-                    f"available_node_types. This node's resources will be "
-                    f"ignored. If you are using an unmanaged node, manually "
-                    f"set the user_node_type tag to \"{NODE_KIND_UNMANAGED}\""
-                    f"in your cloud provider's management console.")
-                return None
+                raise RuntimeError("Missing entry for node_type {} in "
```

Comment:
Was this doing something broken before?

Reply:
just reverted it, no clue how that happened